### ISSUE TITLE
Error while writing CSR server file!

### ISSUE THREAD

**Issue by nazarpleskanka** [LibreSign/libresign#822]
**Describe the bug**
When i try to generate Root certificate data, it is not working
Error which i can see:
`
{success: false, message: "Error while writing CSR server file!"}
message: "Error while writing CSR server file!"
success: false
`
In  nextcloud log:
`
"POST /apps/libresign/api/0.1/admin/certificate HTTP/1.1" 401 821 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36"
162.210.224.2 - tsvetib [26/May/2022:00:23:39 +0300] "PROPFIND /remote.php/dav/files/8D77998D-7
`

**To Reproduce**
- Login to NextCloud
- Go to Settings (/settings/admin/libresign)
- Try to generate Root certificate data

**Expected behavior**
Successfully generated Root certificate

**Screenshots**
<img width="1789" alt="Screenshot 2022-05-26 at 00 30 49" src="https://user-images.githubusercontent.com/67473704/170370849-7de427a5-cc93-4f1f-94e6-aec4ee95e254.png">

**Environment information (please complete the following information):**
 - OS: Docker
 - Browser Chrome
 - LibreSign Version: 	2.4.5
 - Nextcloud Server Version 23.0.2
 - Logs :
 `
162.210.224.2 - - [26/May/2022:00:23:32 +0300] "GET /ocs/v2.php/apps/notifications/api/v2/notifications HTTP/1.1" 304 761 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.0.0 Safari/537.36"
162.210.224.2 - - [26/May/2022:00:23:37 +0300] "POST /apps/libresign/api/0.1/admin/certificate HTTP/1.1" 401 821 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.64 Safari/537.36"
162.210.224.2 - tsvetib [26/May/2022:00:23:39 +0300] "PROPFIND /remote.php/dav/files/8D77998D-72A4-4CD9-B4B3-EDE428FFC2F2/Accounting/Custom%20Clearance HTTP/1.1" 207 1044 "-" "Mozilla/5.0 (Windows) mirall/3.5.0stable-Win64 (build 20220502) (Nextcloud, windows-10.0.17763 ClientArchitecture: x86_64 OsArchitecture: x86_64)"

`

**Additional context**
Nextcloud is running in docker

**Comment by vitormattos** [comment]
Hi, now we have a command to make more easy the setup and download all the binaries necessary to work.

**Comment by nazarpleskanka** [comment]
Hi, could you please provide some details what do you mean here?
How can i fix my problem?
Should i reinstall plugin, which steps i need to run ?
Thanks

**Comment by pwepwe973** [comment]
Hello, thank you for the work you did on this little plugin which can be very useful as soon as it works correctly

but unfortunately I don't understand your answer given to the user who comes to ask for your help! if they come to you, it's because they can't find the solution! I've been on this for 3 days and still haven't found the solution to this problem, because when users come to you, you send them to another page or there is no solution at all, they have to manage on their own etc...

since August 5, 2022 you have snub this user it's really a shame
